src/benchmarks.rs

```diff
@@ -2,29 +2,43 @@
 
 use crate::{BalanceOf, Call, Config, Pallet};
 use cumulus_pallet_parachain_system::Pallet as RelayPallet;
-use cumulus_primitives_core::relay_chain;
-use cumulus_primitives_core::relay_chain::v1::HeadData;
-use cumulus_primitives_core::relay_chain::BlockNumber as RelayChainBlockNumber;
-use cumulus_primitives_core::PersistedValidationData;
+use cumulus_primitives_core::{
+	relay_chain::{v1::HeadData, BlockNumber as RelayChainBlockNumber},
+	PersistedValidationData,
+};
 use cumulus_primitives_parachain_inherent::ParachainInherentData;
 use ed25519_dalek::Signer;
 use frame_benchmarking::{account, benchmarks, impl_benchmark_test_suite};
-use frame_support::dispatch::UnfilteredDispatchable;
-use frame_support::inherent::InherentData;
-use frame_support::inherent::ProvideInherent;
-use frame_support::traits::{Currency, Get}; // OnInitialize, OnFinalize
-use frame_support::traits::{OnFinalize, OnInitialize};
+use frame_support::{
+	dispatch::UnfilteredDispatchable,
+	inherent::{InherentData, ProvideInherent},
+	traits::{Currency, Get, OnFinalize, OnInitialize},
+};
 use frame_system::RawOrigin;
 use parity_scale_codec::Encode;
-use sp_core::crypto::{AccountId32, UncheckedFrom};
-use sp_core::ed25519;
-use sp_core::H256;
-use sp_runtime::traits::One;
-use sp_runtime::MultiSignature;
-use sp_std::vec;
+use sp_core::{
+	crypto::{AccountId32, UncheckedFrom},
+	ed25519,
+};
+use sp_runtime::{traits::One, MultiSignature};
 use sp_std::vec::Vec;
 use sp_trie::StorageProof;
 
+// These is a fake proof that emulates a storage proof inserted as the validation data
+// We avoid using the spoof builder here because it generates an issue when compiling without std
```

Comment:
```suggestion
// This is a fake proof that emulates a storage proof inserted as the validation data
// We avoid using the sproof builder here because it generates an issue when compiling without std
```

If you ever want to diagnose the build issue, I could take a look. But this is fine too.